src/index.html

```diff
@@ -11,12 +11,116 @@
       content="ie=edge"
     />
     <title>Moyo header</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
```

Comment:
The attributes of the <link> tag should each start on a new line with 2-space indentation relative to the tag for better readability.